gnovm/cmd/gno/test.go

```diff
@@ -133,7 +134,7 @@ func execTest(cfg *testCfg, args []string, io *commands.IO) error {
 		cfg.rootDir = guessRootDir()
 	}
 
-	pkgPaths, err := gnoPackagesFromArgs(args)
+	paths, err := gnoPackagePathsFromArgs(args)
```

Comment:
I chose to rename for clarity, we are not dealing with packages here, but paths.